deploy/binding-apiserver/clusterpedia_binding_apiserver_deployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
     spec:
       containers:
         - name: binding-apiserver
-          image: ghcr.io/clusterpedia-io/clusterpedia/binding-apiserver:v0.4.1
+          image: ghcr.io/clusterpedia-io/clusterpedia/binding-apiserver:v0.5.0
```

Comment:
The binding-apiserver does not have version 0.5.0 yet, so we can use the `latest`